packages/slice-machine/lib/models/server/ServerState.ts

```diff
@@ -14,4 +15,5 @@ export default interface ServerState {
   customTypes: ReadonlyArray<CustomTypeSM>;
   remoteCustomTypes: ReadonlyArray<CustomTypeSM>;
   remoteSlices: ReadonlyArray<SliceSM>;
+  clientError: ErrorWithStatus | undefined;
```

Comment:
did `clientError?: ErrorWithStatus` not work?

Reply:
yepp, changed to clientError?: ErrorWithStatus 👍🏼